VotingApplication/VotingApplication.Web/App_Start/BundleConfig.cs

```diff
@@ -43,7 +43,7 @@ public static void RegisterBundles(BundleCollection bundles)
             ngDialogThemeCss.Include("~/Content/Lib/Css/ngDialog-theme-default-min.css");
             bundles.Add(ngDialogThemeCss);
 
-            StyleBundle mdDateTimeCss = new StyleBundle("~/Bundles/StyleLib/mdDateTime", "https://cdn.rawgit.com/SimeonC/md-date-time/v0.0.14/dist/md-date-time.css");
+            StyleBundle mdDateTimeCss = new StyleBundle("~/Bundles/StyleLib/AngularDateTime", "https://cdn.rawgit.com/SimeonC/md-date-time/v0.0.14/dist/md-date-time.css");
```

Comment:
The CDN for this will need to be dropped when I edit the library code so I will fix this (and related comments) when I push that change.